docs/adding_software/overview.md

```diff
@@ -5,3 +5,4 @@
 * [Building software *(for maintainers)*](building_software.md)
 * [Debugging failed builds *(for contributors + maintainers)*](debugging_failed_builds.md)
 * [Deploying software *(for maintainers)*](deploying_software.md)
+* [Support portal *(for contributors + maintainers)*](https://gitlab.com/eessi/support)
```

Comment:
This should point to https://www.eessi.io/docs/support instead

Also, maybe it shouldn't be a bullet point, and a small section at the bottom, like:

```suggestion

If you need help with adding software to EESSI, please [open a support request](support.md).